languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/segment-arabic.srx

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any reason to not have this as part of `segment.srx`, like the other languages?

Reply:
we wanted to isolate the arabic code to avoid conflicts.

Reply:
I might need some help on this one ...

Reply:
Done in https://github.com/languagetool-org/languagetool/pull/2219/commits/b8ae0a60ca1a23bdd7e496dd7771c7869895bb94